hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/mgmt/documentation/DistributionSetsDocumentationTest.java

```diff
@@ -667,4 +665,28 @@ public void createMetadata() throws Exception {
                                 optionalRequestFieldWithPath("[]value")
                                         .description(MgmtApiModelProperties.META_DATA_VALUE))));
     }
+
+    @Test
+    @Description("Invalidates a distribution set. Required Permission: " + SpPermission.UPDATE_REPOSITORY)
+    public void invalidate() throws Exception {
+        final DistributionSet testDS = testdataFactory.createDistributionSet();
+
+        final JSONObject jsonObject = new JSONObject();
+        jsonObject.put("actionCancelationType", "soft");
+        jsonObject.put("cancelRollouts", true);
+
+        mockMvc.perform(post(MgmtRestConstants.DISTRIBUTIONSET_V1_REQUEST_MAPPING + "/{distributionSetId}/invalidate",
+                testDS.getId()).content(jsonObject.toString()).contentType(MediaType.APPLICATION_JSON))
+                .andDo(MockMvcResultPrinter.print()).andExpect(status().isOk()).andDo(
+                        this.document.document(
+                                pathParameters(parameterWithName("distributionSetId")
+                                        .description(ApiModelPropertiesGeneric.ITEM_ID)),
+                                requestFields(
+                                        requestFieldWithPath("actionCancelationType")
+                                                .description(
+                                                        MgmtApiModelProperties.DS_INVALIDATION_ACTION_CANCELATION_TYPE)
+                                                .attributes(key("value").value("['force','soft','none']")),
+                                        requestFieldWithPath("cancelRollouts")
```

Comment:
```suggestion
                                        optionalRequestFieldWithPath("cancelRollouts")
```
`cancelRollouts` is not a mandatory field